src/eduid_webapp/email/views.py

```diff
@@ -247,9 +251,11 @@ def verify_link(user):
             url = urlparse.urlunsplit((scheme, netloc, path, new_query_string, fragment))
             return redirect(url)
         try:
-            verify_mail_address(state, proofing_user)
-            current_app.logger.info('Verified email {} for user {}'.format(email, user))
-            new_query_string = urlencode({'msg': 'emails.verification-success'})
+            if verify_mail_address(state, proofing_user):
+                current_app.logger.info('Verified email {} for user {}'.format(email, user))
+                new_query_string = urlencode({'msg': 'emails.verification-success'})
```

Comment:
I've only looked in the review diff format, so I might be missing something here (but it can also sometimes be good to review parts of code independently of the surrounding code) but how can new_query_string be the appropriate name for something that really seems to be the resulting status code?